docs/en/writing-running-appium/image-comparison.md

```diff
@@ -59,6 +59,67 @@ File.open('match_result_visual.png', 'wb') { |f| f<< Base64.decode64(match_resul
 assert File.size? 'match_result_visual.png'
 ```
 
+```javascript
+Typescript / Javascript
+  /*
+     Typescsript code for occurence comparison using the template matching algorithm.
```

Comment:
I'd say it would be better to add this text to the bottom of the document under a separate heading

Reply:
I had put it in the feature heading. I moved it now to the occurrences heading.

Reply:
Hh you mean the text about  difference between feature and occurrence. I moved it to the top of the occurrence heading.